.gitignore

```diff
@@ -3,3 +3,4 @@ public
 apps/*/translations/*
 !apps/*/translations/src
 .env
+.idea/
```

Comment:
This is more my personal opinion than anything else, but I try to avoid putting developer-specific lines into .gitignore. Every developer working on this repository will end up having stuff in `public` and `translations`, but only those using IntelliJ will have things in `.idea`. A [gobal .gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) might be more appropriate.